understanding/20/parsing.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Parsing</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Authors implement pages properly</dd>
```

Comment:
More of an author task than an objective. Maybe:

```suggestion
		<dt>Objective</dt><dd>Assistive Technologies (AT) can parse web content</dd>
```

Reply:
```suggestion
		<dt>Objective</dt><dd>Assistive Technology can properly present page content</dd>
```